about.html

```diff
@@ -4,25 +4,106 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet">
 </head>
 <body>
+    <header class="top-header">
+        <div class="top-header-items">
+            <div class="social-media-icon">
+                <img src="./res/fb.svg" alt="fb">
+                <img src="./res/twitter.svg" alt="Twitter">
+            </div>
+            <div class="top-header-options">
+                <a href="#">English</a>
+                <a href="#">My Page</a>
+                <a href="#">Log Out</a>
+            </div>
+        </div>
+    </header>
     <nav id="navbar" class="nav-bar">
+
         <div id="hamMenu" class="ham-menu"><img src="./res/mobile_menu.svg" alt="Ham img"></div>
-        <div id="logo"><a href="./index.html"><img src="./res/logo.png" alt="Logo Image"></a></div>
+        <div id="logo"><a href="./index.html"><img src="./res/logo.svg" alt="Logo Image"></a></div>
         <div  class="nav-links nav-link-hide">
-
+            
         </div>
+
     </nav>
-    <section id="aboutMain" class="about-main">
+    <div id="main">
+        <!-- <section id="heroSec" class="hero-sec">
+
+
+        </section>
+        <section id="programDtls" class="program-dtls">
+
+
+        </section> -->
+        <section id="aboutMain" class="about-dtls">
+
+
+
+        </section>
+
+        <section class="about-logo-sec">
+            <div class="about-logo-sec-txt">
+                <h2>GROWTH INTERST <br> GLOBAL SUMMIT <br> 2022</h2>
+                <hr>
+                <p>Market-facing ladder up / ladder back to the strategy build on a culture of contribution and inclusion.</p>
+                <div class="about-logo-img-cont">
+                    <img src="./res/footer-logo.svg" alt="about logo">
+                </div>
+            </div>
+        </section>
+
+        <section class="past-summits">
+            <div class="past-summits-txt">
+                <h3>GROWTH INTERST <br> GLOBAL SUMMIT <br> 2022 Logo</h3>
+                <hr>
+                <p>The website doesn't have the theme i was going for I really like the colour but can you change it there are more projects lined up charge extra the next time.</p>
+            </div>
+            <div class="past-summit-img-cont">
+                <div class="past-summit-img1 past-summit-img">
+                    <img src="./res/aboutgroup1.png" alt="past summit img1">
+                </div>
+                <div class="past-summit-img1 past-summit-img">
+                    <img src="./res/aboutgroup2.png" alt="past summit img1">
+                </div>
+                <div class="past-summit-img2"></div>
+            </div>
```

Comment:
- [ ] Your layout for the section of project if different from the layout we have on the template. Kindly follow the layout on the template tp structure your design.

### Template Layout

![temp-about-layout](https://user-images.githubusercontent.com/26597432/200014358-c35b13b8-3f18-447d-afd4-2f6079593c10.png)


#  Your Layout

![your-about-layout](https://user-images.githubusercontent.com/26597432/200014416-2644e618-fdd9-43c6-bf5d-e9c2fc34b9b5.png)